spec/models/card_spec.rb

```diff
@@ -0,0 +1,53 @@
+require "rails_helper"
+
+RSpec.describe Card, :type => :model do
+  describe Card
+    let(:card){ FactoryGirl.create(:card)}
```

Comment:
Inconsistent indentation detected.<br>Space missing to the left of {.<br>Space missing inside }.